models/spring-ai-openai/src/main/java/org/springframework/ai/openai/api/OpenAiApi.java

```diff
@@ -23,6 +23,7 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonInclude.Include;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import io.micrometer.common.util.StringUtils;
```

Comment:
This should probably be imported from Spring and not Micrometer

`import org.springframework.util.StringUtils;`

Reply:
Actually, it might be an unused import

Reply:
removed